Shared/Win32/Types.cs

```diff
@@ -408,10 +408,21 @@ internal unsafe struct ProcessEntry32
         public string ExeFileName;
     }
 
+    /// <summary>
+    /// Contains process information.
+    /// </summary>
+    /// <remarks>
+    /// Documentation: https://docs.microsoft.com/en-us/windows/desktop/api/winternl/ns-winternl-_peb
+    /// <para/>
+    /// Note that this structure does not attempt to mimic the original C structure field by field, instead
+    /// it relied on offsets based on the original. This is because much of the original strucutre is difficult
+    /// or impossible to reliable define in C# when pointer size and packing of structs can be determined only
+    /// at runtime or during the JITter.
+    /// </remarks>
     [StructLayout(LayoutKind.Sequential)]
     internal unsafe struct ProcessEnvironmentBlock
```

Comment:
Is the size of `struct PEB` not dependent on bitness of the architecture? Is it always 256 bytes?